src/lib.rs

```diff
@@ -302,6 +302,10 @@ impl TwoLineElement {
         Ok(self.elements.epoch())
     }
 
+    pub fn mean_motion(&self) -> f64 {
```

Comment:
Can you give this a `uom` type? I want to avoid raw floats in the public API.

Reply:
Done